src/components/cards/CardContainer.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import CardItem from './CardItem';
+import pasta from '../../assets/beaf_rice.jpg';
+import rice from "../../assets/rice.jpg"
+import NavBar from '../navbar/NavBar';
+
+
+const CardContainer = () => {
```

Comment:
If this is your order component, then it should be named `Order`

Reply:
Sir i named it like this because i think it can be a reusable component since my blog page also needs cards